public/app/features/plugins/admin/components/PluginDetailsPanel.tsx

```diff
@@ -16,73 +29,191 @@ type Props = {
         <Box padding={2} borderColor="medium" borderStyle="solid">
           <Stack direction="column" gap={2}>
-            <Text color="secondary">
-              <Trans i18nKey="plugins.details.labels.links">Links </Trans>
-            </Text>
-            {plugin.details.links.map((link, index) => (
-              <TextLink key={index} href={link.url} external>
-                {link.name}
-              </TextLink>
-            ))}
+            {pluginExtentionsInfo.map((infoItem, index) => {
+              return (
+                <Stack key={index} wrap direction="column" gap={0.5}>
+                  <Text color="secondary">{infoItem.label + ':'}</Text>
+                  <div className={styles.pluginVersionDetails}>{infoItem.value}</div>
+                </Stack>
+              );
+            })}
+            {plugin.updatedAt && (
+              <Stack direction="column" gap={0.5}>
+                <Text color="secondary">
+                  <Trans i18nKey="plugins.details.labels.updatedAt">Last updated:</Trans>
+                </Text>{' '}
+                <Text>
+                  {formatDate(new Date(plugin.updatedAt), { day: 'numeric', month: 'short', year: 'numeric' })}
+                </Text>
+              </Stack>
+            )}
+            {plugin?.details?.lastCommitDate && (
+              <Stack direction="column" gap={0.5}>
+                <Text color="secondary">
+                  <Trans i18nKey="plugins.details.labels.lastCommitDate">Last commit date:</Trans>
+                </Text>{' '}
+                <Text>
+                  {formatDate(new Date(plugin.details.lastCommitDate), {
+                    day: 'numeric',
+                    month: 'short',
+                    year: 'numeric',
+                  })}
+                </Text>
+              </Stack>
+            )}
           </Stack>
         </Box>
-      )}
-
-      {!plugin?.isCore && (
-        <Box padding={2} borderColor="medium" borderStyle="solid">
-          <Stack direction="column">
-            <Text color="secondary">
-              <Trans i18nKey="plugins.details.labels.reportAbuse">Report a concern </Trans>
+        {shouldRenderLinks && (
+          <>
+            <Box padding={2} borderColor="medium" borderStyle="solid">
+              <Stack direction="column" gap={2}>
+                <Text color="secondary">
+                  <Trans i18nKey="plugins.details.labels.links">Links </Trans>
+                </Text>
+                {plugin.url && (
+                  <LinkButton href={plugin.url} variant="secondary" fill="solid" icon="code-branch">
+                    <Trans i18nKey="plugins.details.labels.repository">Repository</Trans>
+                  </LinkButton>
+                )}
+                {plugin.url && (
+                  <LinkButton href={`${plugin.url}/issues/new`} variant="secondary" fill="solid" icon="bug">
```

Comment:
This does not work in cases where `plugin.url` is not the root of the repository - e.g. here https://ephemeral1511182197186ukochka.grafana-dev.net/plugins/mssql (core plugins)

I am not sure if there is much you can do in grafana/grafana besides shifting the logic towards GCOM and handling it the same way you deal with LICENSE urls.